docs/root/intro/deprecated.rst

```diff
@@ -1,64 +1,68 @@
-# DEPRECATED
+.. _deprecated:
+
+Deprecated
+----------
 
 As of release 1.3.0, Envoy will follow a
-[Breaking Change Policy](https://github.com/envoyproxy/envoy/blob/master//CONTRIBUTING.md#breaking-change-policy).
+`Breaking Change Policy <https://github.com/envoyproxy/envoy/blob/master//CONTRIBUTING.md#breaking-change-policy>`_.
 
 The following features have been DEPRECATED and will be removed in the specified release cycle.
 A logged warning is expected for each deprecated item that is in deprecation window.
 
-## Version 1.10.0 (pending)
-* Use of `use_alpha` in [Ext-Authz Authorization Service](https://github.com/envoyproxy/envoy/blob/master/api/envoy/service/auth/v2/external_auth.proto) is deprecated. It should be used for a short time, and only when transitioning from alpha to V2 release version.
+Version 1.10.0 (pending)
+========================
+* Use of `use_alpha` in `Ext-Authz Authorization Service <https://github.com/envoyproxy/envoy/blob/master/api/envoy/service/auth/v2/external_auth.proto>`_ is deprecated. It should be used for a short time, and only when transitioning from alpha to V2 release version.
```

Comment:
So, we have some fields here which should now be `:ref:` back to the proto docs. I think it's not a good use of time to fix all of these up, but maybe you could do it for the few in 1.10.0 and add guidance in the docs above about using proper RST links for fields and messages.